life_expectancy/load_save_data.py

```diff
@@ -0,0 +1,18 @@
+"""All code related with folders and data movimentation"""
+
+from pathlib import Path
+import pandas as pd
+
+DATA_DIR = Path(__file__).parent / 'data'
+
+def load_data() -> pd.DataFrame:
+    """Get data from a tsv"""
+    name_file = DATA_DIR / "eu_life_expectancy_raw.tsv"
+    return pd.read_csv(name_file, sep='\t')
+
+def save_data(df,region) -> None: # pylint: disable=invalid-name
```

Comment:
```suggestion
def save_data(df: pd.DataFrame, region: str) -> None: # pylint: disable=invalid-name
```

Also, why are you silencing that pylint error here?

Reply:
legacy, will change it